modules/vocabularies/src/main/frontend/src/RenderVocabListTable.jsx

```diff
@@ -0,0 +1,165 @@
+//
+//  Licensed to the Apache Software Foundation (ASF) under one
+//  or more contributor license agreements.  See the NOTICE file
+//  distributed with this work for additional information
+//  regarding copyright ownership.  The ASF licenses this file
+//  to you under the Apache License, Version 2.0 (the
+//  "License"); you may not use this file except in compliance
+//  with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+//  software distributed under the License is distributed on an
+//  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//  KIND, either express or implied.  See the License for the
+//  specific language governing permissions and limitations
+//  under the License.
+//
+
+import React from "react";
+
+import {
+  Grid,
+  makeStyles,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+  withStyles
+} from "@material-ui/core";
+
+import Actions from "./Actions";
+
+const Config = require("./config.json");
+const Phase = require("./phaseCodes.json");
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: "100%",
+    marginTop: theme.spacing(3),
+    overflowX: "auto"
+  }
+}));
+
+// Creating customized Table components for a nicer look
+
+const StyledTableRow = withStyles(theme => ({
+  root: {
+    "&:nth-of-type(odd)": {
+      backgroundColor: theme.palette.background.default
+    }
+  },
+}))(TableRow);
+
+const StyledTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.common.white,
+    whiteSpace: "pre"
+  },
+  body: {
+    whiteSpace: "pre",
+    textAlign: "right"
+  },
+}))(TableCell);
+
+export default function RenderVocabListTable(props) {
```

Comment:
Component names should not be verbs.
```suggestion
export default function VocabularyList(props) {
```

Please rename all components in this PR that have verbs in their names.